packages/eslint-config-kibana/package.json

```diff
@@ -22,6 +22,7 @@
     "eslint": "^4.1.0",
     "eslint-plugin-babel": "^4.1.1",
     "eslint-plugin-import": "^2.6.0",
+    "eslint-plugin-jest": "^20.0.3",
```

Comment:
This will always require the jest eslint plugin be installed, but I think that's okey. I don't see any harm in people not using Jest having to install this, as this works great for Kibana and xpack.